.github/workflows/tests.yaml

```diff
@@ -0,0 +1,21 @@
+name: Test
+on:
+  push:
```

Comment:
Recommend just:

```
on: push, pull_request
```

What we want is to run tests on every push and PR, yeah?

Reply:
oh yeah